runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -126,8 +132,11 @@ public CheckpointWriter(CorfuRuntime rt, UUID streamId, String author, T map) {
      */
     public Token appendCheckpoint() {
         long start = System.currentTimeMillis();
```

Comment:
Actually let's get rid of the marker, its not needed you can just use the stream tail.